README.md

```diff
@@ -5,28 +5,102 @@ This guide is provided as is, it may not work with everyone's Linux setup and ma
 Because we're not Linux experts, we cannot guarantee to be able to help you fix your issues. You may check [our Discord](https://thepeacockproject.org/discord) in the Linux megathread help channel.
 We're also open to pull requests and tips on how to improve this guide!
 
-# Setup H3 for Peacock with Steam on Linux (and Steam Deck)
+# Setup
 
 Instructions:
-- Clone this git repository
-- Download and extract Peacock in the `linux-steam-setup/Peacock` folder
-    - The `chunk0.js` file must be available at the following path `linux-steam-setup/Peacock/chunk0.js`
-- Run the `install-node.sh` script to install Node for Peacock
-- Edit HITMAN 3 launch options in Steam to `PROTON_LOG=1 PROTON_DUMP_DEBUG_COMMANDS=1 %command%`
-- Start and close the game's launcher from Steam
-- Go to the game install location, folder should contain the Launcher.exe
-    - Copy the `PeacockPatcher.exe` from inside the Peacock folder and the `WineLaunch.bat` from `linux-steam-setup` and paste them into the game's install location.
-- Copy the `run` file from `/tmp/proton_USERNAME/` into `linux-steam-setup` folder
-- Open the `run` file with your favorite editor, locate the line starting with `DEF_CMD` and replace `Launcher.exe` to `WineLaunch.bat`
-- Click the `Hitman 3 (Peacock)` shortcut to start the game with Peacock
-- Edit Peacock Patcher Server address to `127.0.0.1:3000`
-- Game is ready to play with Peacock
-
-## Start it from Steam
-
-- Add your Terminal app as a non-Steam game
-    - Use Konsole for the Steam Deck
-- Open the shortcut properties and change the following:
-    - `Start in` to the folder where the `start.sh` script is. (eg: `"/home/deck/linux-steam-setup/"`, quotation marks are necessary)
-    - `Launch options` to the following `-e "./start.sh"` (quotation marks are necessary)
-- You will now be able to launch Peacock from Steam (or Game Mode on the Steam Deck)
+## Ensure you have the dependencies
+  - `curl`
+  - `unzip`
+
+## Clone this git repository
+
+```
+git clone https://github.com/thepeacockproject/linux-steam-setup
+```
+
+Downloading the zip works too, if you don't have git installed
+
```

Comment:
Some extra instructions are required to specify that the `linux-steam-setup` folder must be called that way (when you download as a zip it names the folder `linux-steam-setup-master`) and must be put at the root of your home folder.
Maybe an extra note about how to edit the `peacock.service` file to change the folder path could be useful too

Reply:
The name of the folder doesn't matter, but yes I'll add a note to edit the service file